cur-test/cur/tests/ntac/software-foundations/Poly-abbrv.rkt

```diff
@@ -44,8 +44,8 @@
   (∀ [n : nat] [m : nat]
```

Comment:
This file intentionally tests the implicit-naming versions of various tactics (that's what the "abbrv" in the file name means).

I agree that continually having to update the file is annoying so I'm open to suggestions, but I've tried to restrict these kinds of tests to this one file (I think?) so at least the damage is minimized.

But using named versions of the tactics makes this test file redundant.

Reply:
I will probably leave out this commit, if that is ok.

Reply:
Oh okay that's fine.

I'll consider how to test the generated names before merging into master.